Source/OxyPlot/Rendering/OxyAnnulus.cs

```diff
@@ -0,0 +1,207 @@
+﻿// --------------------------------------------------------------------------------------------------------------------
+// <copyright file="OxyCircle.cs" company="OxyPlot">
+//   Copyright (c) 2014 OxyPlot contributors
+// </copyright>
+// <summary>
+//   Describes the width, height, and point origin of a rectangle.
+// </summary>
+// --------------------------------------------------------------------------------------------------------------------
+
+namespace OxyPlot
+{
+    using System;
+    using System.Globalization;
+    using System.Text;
+
+    /// <summary>
+    /// Describes the center, the inner and the outer radius of a annulus.
+    /// </summary>
+    public struct OxyAnnulus : IOxyRegion, IEquatable<OxyAnnulus>, IFormattable
+    {
+
+        /// <summary>
+        /// Gets the center point of the annulus.
+        /// </summary>
+        /// <value>The center.</value>
+        public ScreenPoint Center { get; set; }
+
+        /// <summary>
+        /// Gets the inner radius of the annulus
+        /// </summary>
+        public double InnerRadius { get; set; }
+
+        /// <summary>
+        /// Gets the outer radius of the annulus
+        /// </summary>
+        public double OuterRadius { get; set; }
```

Comment:
These should be getter only (i.e. `{ get; }`)

Reply:
changed to private setters

Reply:
That's ok for me. I'm used to do it that way. Maybe something I should think about... Refactoring maybe takes longer in some cases, but provides some not neglectable safety.
Maybe I'll use this concept more in the future.